dojo/templates/dojo/view_finding.html

```diff
@@ -473,48 +473,64 @@ <h4>Similar Findings<span class="pull-right"><a data-toggle="collapse" href="#si
                         <tr>
-                            <th>Finding</th>
+                            <th>Title</th>
                             <th>Status</th>
-                            <th>Tags</th>
+                            <th>Date</th>
                             <th>Test</th>
                             <th>Engagement</th>
-                            <th>Version</th>
+                            {% if finding.cwe > 0 %}
+                            <th>CWE</th>
+                            {% endif %}
                             <th>CVE</th>
                             <th>File</th>
                             <th>Deduplicate?</th>
                         </tr>
                         {% for similar in findings %}
                             <tr>
+                                <td>
+                                    <a title="{{ similar.title }}" href="{% url 'view_finding' similar.id %}">{{ similar.title|truncatechars_html:80 }}</a>
+                                    {% if similar.tags %}
+                                    <small>
+                                        {% for tag in similar.tags %}
+                                            <a title="Search {{ tag }}" class="tag-label tag-color"
+                                               href="{% url 'simple_search' %}?query={{ tag }}">{{ tag }}</a>
+                                        {% endfor %}
+                                    </small>
+                                    {% endif %}
+                                </td>
+                                <td>{{ similar.status }}{% if similar.duplicate_finding_id %}, <a href="{% url 'view_finding' similar.duplicate_finding_id %}">Original</a> {% endif %}</td>
                                 <td>
                                     <a target="#" data-toggle="tooltip" data-placement="bottom" title="{{ similar.title }}" href="{% url 'view_finding' similar.id %}">{{ similar.date }}</a>
                                     {% with similar.notes.count as note_count %}
                                         ({{ note_count }} note{{ note_count|pluralize }})
                                     {% endwith %}
                                 </td>
-                                <td>{{ similar.status }}{% if similar.duplicate_finding_id %}, <a href="{% url 'view_finding' similar.duplicate_finding_id %}">Original</a> {% endif %}</td>
+                                <td><a href="{% url 'view_test' similar.test_id %}">{{ similar.test.test_type.name }}</a></td>
+                                <td><a href="{% url 'view_engagement' similar.test.engagement_id %}">{{ similar.test.engagement.name }}</a>
+                                {% if similar.test.engagement.version%}
+                                    <sup>
+                                          <a target="_blank" class="tag-label tag-version" data-toggle="tooltip" data-placement="bottom" title="Product Version: {{ eng.version }}">
```

Comment:
The trailing `:` is probably not needed here.

Reply:
I don't see it :-)

Reply:
Ah yes.. well, that's bc in my test it shows `Product Version:` and the `eng.version` is empty.. so it looks weird.

Reply:
I think that's maybe because if the %} without a space earlier. The whole title thing shouldn't be displayed if there's no version.

Reply:
ah eng.version was wrong, fixed in new commit